### ISSUE TITLE
search in phonetics

### ISSUE THREAD

**Issue by eroux** [buda-base/public-digital-library#943]
Now that we've decided on the phonetics route, there are a few things to consider:

The first fundamental question for me is to choose between two routes:
- the first route is to create a sort of standard BDRC phonetics and add it to the records, so that when people search in phonetics they find results, and they see the phonetics that matches their request
- the second route is that when users search in phonetics they get results in Wylie or Unicode, but don't see the internal phonetics

This is a fundamental UX decision that will drive the next steps of the developments. I would argue the best solution is the second one, with the following arguments:
- this is a solution that can also be used for a lenient search in Tibetan Unicode, while having some English phonetics will be useless for that
- there is a lot of emotions and conflicting habits in people's relationship with phonetic system and there will be a lot of complains that the phonetics should be this and that instead of what we display
- this is a more flexible system as we don't have to find a phonetics that looks nice and makes sense, the only requirement is that it helps the search
- automatic word segmentation is still not a solved problem for Tibetan (especially in verse and titles), and a syllable based on syllables will look a bit silly

what do you think @roopeux ?

**Comment by roopeux** [comment]
My reasoning followed pretty much the same path as yours and I am in favour of the second option.

As you indicated, users don't naturally type phonetics in syllables and we cannot make them learn our phonetics.  Moreover, it is very useful to have full words in queries, because it rids us from two-phrase guessing.

**Comment by eroux** [comment]
good point about the grouping into words by the user through the phonetic!

**Comment by eroux** [comment]
Just to connect with some existing issues on the search engine side: https://github.com/buda-base/lucene-bo/issues/48
